### ISSUE TITLE
update atom like redux

### ISSUE THREAD

**Issue by TBSten** [pmndrs/jotai#1020]
I want to update jotai atom without using useAtom or useSetAtom.

 So I want to write code like this:

 ```jsx


 const counterAtom = atom (0);


 // in component


 counterAtom.set (1);


 ```

 This is to replace the redux code below.

 ```jsx


 const store = ...;


 // in component


 store.dispatch (setCount (1));


 ```

 Is there any good way?

**Comment by dai-shi** [comment]
jotai is modeled after `useState` React state, which does not exist outside React tree. So, conceptually, it doesn't work like global states that can accessed outside React, like redux, zustand and valtio.

That said, due to the high demand, we are experimenting a new API. See #861 #922. Available in v1.6.0 as an unstable feature.

To do what you want in stable features, you need a global state outside React and integrate it into jotai. Some of integrations may help: `jotai/valtio`, `jotai/zustand`, `jotai/redux` and `atomWithObservable`.

**Comment by TBSten** [comment]
thank you good answer ! 
I will use jotai/valtio, jotai/zustand, jotai/redux and atomWithObservable .
